cnd/src/storage.rs

```diff
@@ -198,23 +197,23 @@ impl Load<respawn::Swap<herc20::Params, hbit::Params>> for Storage {
             Role::Bob => secret_hash.ok_or_else(|| NoSecretHash(id))?.secret_hash.0,
         };
 
-        let swap = respawn::Swap {
-            id,
+        let swap = network::start_swap::Swap {
             role,
             alpha: build_herc20_params(herc20, secret_hash, id)?,
             beta: build_hbit_params(self.seed, hbit, id, role, secret_hash)?,
+            start_of_swap: swap.start_of_swap,
         };
 
         Ok(swap)
     }
 }
 
 #[async_trait::async_trait]
-impl Load<respawn::Swap<hbit::Params, herc20::Params>> for Storage {
+impl Load<network::start_swap::Swap<hbit::Params, herc20::Params>> for Storage {
```

Comment:
I think I would fit better if the `start_swap` module would be outside of `network` :)